index.css

```diff
@@ -43,39 +62,33 @@ body {
   padding-right: 35px;
 }
 
-.menu_bar {
-  width: 24px;
-  height: 24px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  display: flex;
-  justify-content: center;
-}
-
-.menu_bar img {
-  width: 18px;
-  height: 12px;
-  left: 3px;
-  top: 6px;
-  align-self: center;
-}
-
 .main_container {
   top: 92px;
   background: #1c1a19;
-  background-image: url('img/Illustration-header.png');
-  background-position-x: -50%;
-  background-position-y: -20%;
-  background-repeat: no-repeat;
   gap: 12px;
+  height: 100vh;
 }
 
 .intro {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   align-items: flex-start;
-  padding: 120px 24px;
+  height: 100vh;
+}
+
+.intro_item1 {
+  order: 1;
+  margin: 20% 5% 0 5%;
+}
```

Comment:
- [ ] On mobile screen starting from `401px` your design doesn't match the mobile Figma design. Kindly fix this 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206722029-a0cb9917-bbd8-4477-86f7-5e2b630f4b55.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206721991-181e7381-1f11-428f-bf15-6b561f70b702.png)

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206722704-b442091c-5b28-4dfe-a1fe-fd47831e8f35.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206722504-5ba741f1-8456-4114-8f8d-6a3e94ea5479.png)